tests/Integration/Core/Translation/Provider/OthersCatalogueLayersProviderTest.php

```diff
@@ -211,4 +203,29 @@ private function getProvider(array $databaseContent = []): CoreCatalogueLayersPr
             $providerDefinition->getTranslationDomains()
         );
     }
+
+    /**
+     * @param array $expected
+     * @param MessageCatalogue $catalogue
+     */
+    private function assertResultIsAsExpected(array $expected, MessageCatalogue $catalogue): void
```

Comment:
@sowbiba it's not that important but I see that you use this method in every `Provider` test, right? what about creating a class with this method and rest of the duplicated code? 🤔 from what I see you can set expected array and omit having `testItLoadsCustomizedTranslationsWithNoThemeFromDatabase` in every file 🤔 

but those are tests, I'm not sure how important here is to avoid duplications 👍 

those files are almost identical in many places

Reply:
I'm hesitating to add an Abstract or an Helper with this method in it. But I dont know if it's tests friendly. I will give it a try